### PR TITLE
Two new questions about sql injection security.alvaro tab

### PR DESCRIPTION
**Pull request by raykotab**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -43,6 +43,27 @@ const securityQuiz = [
       "White Hat Hackers work for organizations as Cybersecurity Engineers, Penetration Testers, etc.  where they perform tasks like ethically hacking the system to find and report vulnerabilities. ",
     Link: "https://www.freecodecamp.org/news/white-hat-black-hat-red-hat-hackers/",
   },
+  {
+    Question: "What is a common exploited Web security vulnerability",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:47]:
```suggestion
    Question: "What is a common exploited Web security vulnerability?",
```

File: src/data/security-quiz.ts [hunk 2]
```diff
@@ -43,6 +43,27 @@ const securityQuiz = [
...
+  },
+  {
+    Question:
+      "What is the first line of defense against SQL injection attacks?",
+    Answer: "Sanitizing and validating user inputs",
+    Distractor1: "Good Firewall setup",
+    Distractor2: "IP white- and blacklist",
+    Distractor3: "Good Antivirus",
+    Explanation:
+      "Sanitizing your user input is a basic common rule in security, make sure not to add directly GET or POST variables in your code without a proper sanitizing function. ",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:64]:
```suggestion
      "Sanitizing your user input is a basic common rule in security. It is important to not directly add GET or POST variables in your code without a proper sanitizing function. ",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -333,6 +333,17 @@ const sqlQuiz = [
...
+    Question:
+      "In SQL, what is the command used to give a record a date and time value from the system on execution?",
+    Answer: "NOW()",
+    Distractor1: "TIMESTAMP()",
+    Distractor2: "Y-M-D()",
+    Distractor3: "DATETIME()",
+    Explanation:
+      "The NOW() function gives the selected record a value of the current timestamp",
+    Link: "https://www.freecodecamp.org/news/sql-date-functions-getdate/",
+  },
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:346]: This question was already added in your last PR.
It looks like maybe you are working with an outdated branch.
I would suggest updating your branch so you are working with the latest versions of the quiz questions.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 3 above)
Comment (raykotab) [reply src/data/sql-quiz.ts:346]: Hi! @jdwilkin4, I am sorry, but I am unsure what to do now with this not-properly updated code piece on the SQL file. I tried to update my fork and pull on my main, and merge to my branch, but it feels a bit complicated to solve this conflict. Could you please help me out?

Comment (jdwilkin4) [reply src/data/sql-quiz.ts:346]: Sure, I can take a look at it after I wrap up this PR for work 👍

Comment (raykotab) [reply src/data/sql-quiz.ts:346]: Thanks!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by raykotab** [comment]
ok, so my try seems not so far fetched, thanks @jdwilkin4, may I request the new review?

**Pull request merged**